src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public Figure createRandomFigure() {
+        return new Circle(getRandomInt());
+    }
```

Comment:
The `createRandomFigure` method should not be in the `Circle` class. Consider moving the logic for creating random figures to a separate class like `FigureSupplier`.